src/__tests__/config/translations.test.ts

```diff
@@ -1,7 +1,45 @@
+import Path from 'path'
+import fs from 'fs'
 import translations from '../../config/localization/translations.json'
 
 describe('Check translations integrity', () => {
   it.each(Object.keys(translations))('Translation key value should be equal', (key) => {
     expect(key).toEqual(translations[key])
   })
 })
+
+describe('Check translations available', () => {
+  const files = []
+  const translationKeys = Object.keys(translations)
+
+  function throughDirectory(directory) {
+    fs.readdirSync(directory).forEach((file) => {
+      const absolute = Path.join(directory, file)
+      if (fs.statSync(absolute).isDirectory()) return throughDirectory(absolute)
+      if (absolute.includes('.tsx')) {
+        return files.push(absolute)
+      }
+      return files.length
```

Comment:
Why are we returning `files.length`?

Reply:
The return above returns new array's length so I keep the consistency although that return is not used at the end